python/paddle/v2/framework/layers.py

```diff
@@ -58,8 +58,9 @@ def fc(input,
     return helper.append_activation(pre_activation)
 
 
-def embedding(input,
+def embedding(Ids,
               size,
+              name="default_embedding",
```

Comment:
We need to ensure the parameter name is the same, not the embedding layer name is the same.